src/java.base/share/classes/java/io/File.java

```diff
@@ -1421,7 +1435,9 @@ public boolean mkdirs() {
     }
 
     /**
-     * Renames the file denoted by this abstract pathname.
+     * Renames the file denoted by this abstract pathname.  If this pathname
+     * denotes a symbolic link, then the link itself, not its target, will be
+     * renamed.
```

Comment:
For FIS/FOS/RAF it's just two constructors in each that follow links so I'm wondering if might be better to put the text there. Also in the case of FOS (and RAF when "w" is in the mode) it will follow the linking when creating (or truncating in the case of FOS) the file and part of wonders if we need to say more on this.

Reply:
> or truncating in the case of FOS

For that matter, I don't see where truncation is explicitly mentioned in the FOS doc.

Reply:
These comments are addressed in f492c36.